.github/workflows/actionlint.yml

```
@@ -0,0 +1,22 @@
+name: Lint GitHub Actions workflows
+
+on:
+  push:
+    paths:
+      - ".github/workflows/*.ya?ml"
+  pull_request:
+    paths:
+      - ".github/workflows/*.ya?ml"
+
+jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download actionlint
+        id: get_actionlint
+        run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/2ab3a12c7848f6c15faca9a92612ef4261d0e370/scripts/download-actionlint.bash)
+        shell: bash
+      - name: Check workflow files
+        run: ${{ steps.get_actionlint.outputs.executable }} -color -ignore 'SC2086|SC2016'
```

Let's document what the suppressed rules are, to make it easier for future maintenance.
```suggestion
        # Ignored shellcheck rules:
        # - https://www.shellcheck.net/wiki/SC2086 - "Double quote to prevent globbing and word splitting."
        # - https://www.shellcheck.net/wiki/SC2016 - "Expressions don't expand in single quotes, use double quotes for that."
        run: ${{ steps.get_actionlint.outputs.executable }} -color -ignore 'SC2086|SC2016'
```

Btw, what happens if we don't suppress `SC2086`? That rule looks useful _in general_ so I'm wondering what it's flagging here.

---

> I agree with "$GITHUB_OUTPUT" 

The same as this. Sorry my mistake here, was fighting with my yaml formatter and ignored these two it is😔

---

No worries, mistakes happen and learning from them is what's important :) Thanks for all the work you've put into cargo-semver-checks already!